search/src/main/java/com/yas/search/config/ServiceUrlConfig.java

```diff
@@ -0,0 +1,8 @@
+package com.yas.search.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "yas.services")
+public record ServiceUrlConfig(
+        String media) {
```

Comment:
I added this just in case for now, we might need it to get product thumbnail image, however according to anh Thiện, we might get the image in frontend application. So we might get rid of this once the solution is decided.